pip_audit/_dependency_source/resolvelib/pypi_provider.py

```diff
@@ -465,6 +500,8 @@ def is_satisfied_by(self, requirement: Any, candidate: Any) -> bool:
         """
         See `resolvelib.providers.AbstractProvider.is_satisfied_by`.
         """
+        if isinstance(candidate, SkippedCandidate):
+            return True
```

Comment:
Nit: IMO we should have a small comment here and in `get_dependencies` explaining why we do this (`SkippedCandidate` already explains this, so maybe just a ref telling the reader to look at the docstring there?)